packages/lime-system/files/etc/config/lime-defaults

```diff
@@ -24,6 +24,8 @@ config lime network
 	list resolvers 4.2.2.2   # b.resolvers.Level3.net
 	list resolvers 141.1.1.1 # cns1.cw.net
 	list resolvers 2001:470:20::2 # ordns.he.net
+	option bmx6_over_batman false
+	option bmx7_over_batman false
```

Comment:
When you run lime-config it will print warnings on the shell if some default is missing